OpenSim/Tools/IMUInverseKinematicsTool.cpp

```diff
@@ -200,7 +222,10 @@ void IMUInverseKinematicsTool::runInverseKinematicsWithOrientationsFromFile(
 
     std::cout << "Wrote IK with IMU tracking results to: '" <<
         outputFile << "'." << std::endl;
-
+    if (get_report_errors()) {
+        STOFileAdapter_<double>::write(*modelOrientationErrors,
+                get_results_directory() + "/" + "orientationErrors.sto");
```

Comment:
I think the user-given name of the tool should be included in the filename.

Reply:
Done, thanks for the feedback @chrisdembia